Source: Gentoo
Upstream: Unknown
Reason: Fixes test failures
	depcomp2.test: Newer autoconf, Gentoo bug #181484.
	man2.test: Newer autoconf, Gentoo bug #159557.

--- automake-1.5/tests/depcomp2.test
+++ automake-1.5/tests/depcomp2.test
@@ -16,6 +16,7 @@
 
 cat > Makefile.am << 'END'
 SUBDIRS = subdir
+datarootdir = @datarootdir@
 END
 
 rm -f depcomp

--- automake-1.6.3/tests/man2.test
+++ automake-1.6.3/tests/man2.test
@@ -23,7 +23,7 @@
 $AUTOMAKE

 # Let's play with $DESTDIR too, it shouldn't hurt.
-./configure --prefix=''
+./configure --prefix='' --mandir=/man
 DESTDIR="`pwd`/=inst" $MAKE -e SHELL=/bin/sh install

 test -f ./=inst/man/man2/foo.2
